### DISCUSSION TITLE
Default Lobby Settings

### DISCUSSION THREAD

**Discussion by WilliamWeberBerrutti** [OpenHV/OpenHV#857] [Ideas]
What if the default lobby settings had Limit 'Build Area: **disabled**' and 'Starting Units: **Base + Scout**'?

**Comment by Mailaender** [comment]
Unlimited build area remove the need for outpost and add base creep which can be problematic. https://github.com/OpenHV/OpenHV/pull/812 shows the map by default which makes scout less vital.

**Comment by Rampoina** [comment]
the scout is still very important in 1v1, there's builds that can come fairly fast that are hard to scout otherwise

**Comment by WilliamWeberBerrutti** [comment]
Outpost can still be useful on large maps. I usually use them to build extra storages, turrets and power plants on the map's corners (or on distant points of interest).

**Comment by Rampoina** [comment]
Outposts can also be used aggressively by building a production facility on the front and thereby removing defender's advantage this way (you only need to build 1 too, no need to build them all on the front)
